dist/index.html

```diff
@@ -145,18 +146,18 @@ <h2 class="heading--secondary">What is Codidact?</h2>
             <div class="content-wrapper">
                 <h3 class="heading--secondary">Community First</h3>
                 <p>
-                    Our goal is to build an open-source platform for community-driven Q&A sites. Our aim is to keep it
-                    accessible for beginners and newcomers to the community while optimized for long-time users,
-                    focusing around the answer. We'll fund it by seeking donations, not by paywalls, and work with
-                    individual communities that want to join us.
+                    Our goal is to build an open-source platform for community-driven Q&A sites. 
+                    We aim to welcome beginners and experts alike and to empower communities of learners and teachers. 
+                    We'll fund it by seeking donations, not by paywalls, and work with individual communities that want to join us.
                 </p>
             </div>
         </section>
         <section class="row content-wrapper call-to-action">
             <p>
-                Ultimately, we are all looking for answers to life's big - and little questions. We want to build an
-                amazing Q&A platform that makes it easier to ask and curate questions in a non-hostile environment and
-                get to the answers you need, without the fuss.
+                Ultimately, we are all looking for answers to life's big - and little - questions. 
```

Comment:
Shouldn't this be either `big &ndash; and little &ndash; questions.` or `big&mdash;and little&mdash;questions.` rather than hyphens (&ndash; &mdash;)?

At least according to https://www.punctuationmatters.com/hyphen-dash-n-dash-and-m-dash/.

Reply:
It should, yeah.  I kept meaning to mention that and kept forgetting.  I'm personally fine with -- (though HTML *does* have codes for this, so I guess we should).  We shouldn't use single hyphens.

These were already there, so it's not this PR's fault.  We could merge and fix that separately or do one more round here, whatever our git policymakers say.

Reply:
yeah lets make the change now - @CharlieBrumbaugh  if you can squash to the last commit then will merge

Thanks for this 👍